### PR TITLE
Adapt to akkaserverless new samples

### PR DESCRIPTION
**Pull request by octonato**
This makes the UI compatible with the new samples we added for Akka Serverless (ongoing PRs https://github.com/lightbend/akkaserverless-java-sdk/pull/10 and https://github.com/lightbend/akkaserverless-java-sdk/pull/12).

### CODE REVIEW

File: frontend/shoppingcart.proto [hunk 1]
```diff
@@ -36,26 +34,8 @@ message Cart {
...
-        option (google.api.http).post = "/com.example.shoppingcart.ShoppingCart/cart/{user_id}/items/{product_id}/remove";
-    }
-
-    rpc GetCart(GetShoppingCart) returns (Cart) {
-        option (google.api.http) = {
-          get: "/com.example.shoppingcart.ShoppingCart/carts/{user_id}",
-          additional_bindings: {
-            get: "/carts/{user_id}/items",
-            response_body: "items"
-          }
```
Comment (octonato) [inline frontend/shoppingcart.proto:58]: remove the annotations since we are using grpc-web and not REST calls

File: frontend/shoppingcart.proto [hunk 2]
```diff
@@ -1,29 +1,27 @@
 // This is the public API offered by the shopping cart entity.
-syntax = "proto3";
 
-import "google/protobuf/empty.proto";
-import "cloudstate/entity_key.proto";
-import "cloudstate/eventing.proto";
-import "google/api/annotations.proto";
```
Comment (octonato) [inline frontend/shoppingcart.proto:7]: no extra annotations needed

File: frontend/shoppingcart.proto [hunk 3]
```diff
@@ -1,29 +1,27 @@
...
 
 package com.example.shoppingcart;
 
-option go_package = "tck/shoppingcart";
+option java_outer_classname = "ShoppingCartApi";
+
+import "google/protobuf/empty.proto";
 
 message AddLineItem {
-    string user_id = 1 [(.cloudstate.entity_key) = true];
```
Comment (octonato) [inline frontend/shoppingcart.proto:14]: clients should not depend on cloudstate/akkaserverless annotations

**Review by pvlugter — APPROVED** [review]
LGTM

**Review by katsutoxin — APPROVED** [review]
LGTM!

**Pull request merged**